### PR TITLE
Add IO Zoom

### PR DESCRIPTION
**Pull request by iozoom**
Hi,

Please add the iozoom website to  two factor authentication  list. Changes are already made in the yml files.

### CODE REVIEW

**Review by mxxcon — CHANGES_REQUESTED** [review]
Travis validation fails with these errors:
1. _data/hosting.yml is not alphabetized by name
2. 'twitter' should NOT be present when tfa: true.
3. 'facebook' should NOT be present when tfa: true.

Please address them.
Also `exception` tag is not really needed, since it doesn't say anything out of ordinary.

**Comment by iozoom** [comment]
changes made as you suggested.

File: _data/hosting.yml [hunk 1]
```diff
@@ -173,6 +173,13 @@ websites:
       twitter: inmotionhosting
       img: inmotion.png
       tfa: No
+      
+     - name: IO Zoom
```
Comment (mxxcon) [inline _data/hosting.yml]: In `yaml` indentation/spaces are important. You indented this line a bit too much.

**Comment by iozoom** [comment]
I corrected the space. Please review

File: _data/hosting.yml [hunk 2]
```diff
@@ -173,6 +173,13 @@ websites:
       twitter: inmotionhosting
       img: inmotion.png
       tfa: No
+    
+     - name: iozoom
```
Comment (mxxcon) [inline _data/hosting.yml]: Still 1 extra space 😉 
I'm not picky, it's just indentation is critical to yaml format and without everything being correct, the site won't generate the new changes.

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Comment by iozoom** [comment]
I deleted that line. Please verify.

**Review by mxxcon — APPROVED** [review]

**Comment by mxxcon** [comment]
@iozoom since you were still having problems there, I fixed it up for you here https://github.com/2factorauth/twofactorauth/pull/2508/commits/b6ac34d14ccf1663febe6f2285b0e87841f1ae8d

**Review by jamcat22 — APPROVED** [review]
I corrected the name to reflect the website's branding.
Looks good to me!

**Comment by iozoom** [comment]
okay, thanks, Do I need to do anything else to publish the website under https://twofactorauth.org/?ref=producthunt#hosting

**Comment by mxxcon** [comment]
See https://github.com/2factorauth/twofactorauth/wiki/FAQ_Merge

**Comment by iozoom** [comment]
Okay. so I just wait right ?

**Comment by mxxcon** [comment]
Yup, just wait.

**Pull request merged**